api/src/test/java/org/openmrs/module/fhir/api/LocationServiceTest.java

```diff
@@ -143,4 +143,18 @@ public void updateLocationById_shouldAddResourceIfNotExistsElseUpdateExistingRes
 		}
 	}
 
+	/**
+	 * @verifies create omrs location
+	 */
+	@Test
+	public void createOpenMRSLocation() throws Exception {
+		String locationUuid = "6f42abbc-caac-40ae-a94e-9277ea15c125";
+		org.openmrs.Location omrsLocation=Context.getLocationService().getLocationByUuid(locationUuid);
+		omrsLocation.setUuid("");// remove the uuid value from the Location. This will let this resource to be persist on the db with
+		// random uuid
+		Location fhirLocation = FHIRLocationUtil.generateLocation(omrsLocation);	
+		fhirLocation.setName("New Location Test Name");
+		fhirLocation = Context.getService(LocationService.class).createLocation(fhirLocation);
+		assertNotNull(fhirLocation);
```

Comment:
In a test method, asserting not null only not sufficient. Make sure you assert some attributes as well.